.circleci/config.yml

```diff
@@ -25,6 +25,10 @@ jobs:
                 command: ./node_modules/.bin/eslint src
                 when: always
 
+            - run:
+                name: Build node.js package
+                command: npm run build:js
```

Comment:
```suggestion
                command: npm run build:all
```

Reply:
This doesn't run (look at the commit history of this PR), which is expected, since the node environment doesn't have any python deps...